.gitignore

```diff
@@ -1,4 +1,3 @@
 kata-agent
 kata-agent.service
-agent
```

Comment:
maybe this should be a `/agent`, to just ignore the agent file in the top level dir?
tbh, I don't immediately see how it would be affecting anything under `vendor`, unless there were a file just called `agent` in there - but, prefixing with `/` I'm thinking might be the right thing to do?

Reply:
As a general rule, we should be ignoring as few files as possible. But that entry was causing problems as it is bogus (now). It may have been valid once but retaining it causes problems for files such as:

```
vendor/github.com/uber/jaeger-client-go/thrift-gen/agent/agent.go
```

Reply:
right - adding the `/` on the front should make it say 'just the agent file in the top level dir' afaict - which feels like what the original intent was.

Reply:
That makes sense. But we don't need that rule any more ;)